packages/contracts/tests/js/StabilityPoolTest.js

```diff
@@ -3982,6 +3982,36 @@ contract('StabilityPool', async accounts => {
       const txD = await stabilityPool.registerFrontEnd(dec(1, 18), { from: D });
       assert.isTrue(txD.receipt.status);
     });
+    
+    it("setCommunityIssuanceAddress(): reverts if address is invalid", async() => {
+      // zero address
+      await th.assertRevert(stabilityPool.setCommunityIssuanceAddress(ZERO_ADDRESS, {from: owner}), "Account cannot be zero address");
+
+      // non-contract address
+      await th.assertRevert(stabilityPool.setCommunityIssuanceAddress(C, {from: owner}), "Account code size cannot be zero");
+    })
+
+    it("setCommunityIssuanceAddress(): reverts if initiate by non-authorized user", async() => {
+      // non-owner
+      await th.assertRevert(stabilityPool.setCommunityIssuanceAddress(contracts.defaultPool.address, {from: E}), "Ownable:: access denied");
+    })
+
+    it("setCommunityIssuanceAddress(): should set the new community issuance contract", async() => {
+      const oldCommunityIssuanceAddress = await stabilityPool.communityIssuance();
```

Comment:
despite this is technically correct, it would be more clear and future-proof to deploy and set a new ComminityIssuance contract address